src/Data/V/Internal/Linear/V.hs

```diff
@@ -0,0 +1,166 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE ConstraintKinds #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE KindSignatures #-}
+{-# LANGUAGE LinearTypes #-}
+{-# LANGUAGE MagicHash #-}
+{-# LANGUAGE NoImplicitPrelude #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE TypeOperators #-}
+{-# LANGUAGE UnboxedTuples #-}
+{-# LANGUAGE UndecidableInstances #-}
+module Data.V.Internal.Linear.V
```

Comment:
I've actually been doing `Internal.Linear` in all my module organization PRs. It made sense to me as a way to put all internal modules in one place, even if they aren't `Linear`.

Reply:
Name hierarchies in Haskell are rather unintuitive to me. But I think that `XXX.Linear.Internal` is more consistent with the rest of Haskell. Since what we really have is a module named `Foo.Bar` and its internal module behind `Foo.Bar.Internal`.

Worth changing for consistency.